### ISSUE TITLE
During building : from tex to pdf => l.216 \tightlist

### ISSUE THREAD

**Issue by AnasGhrab** [openframeworks/ofBook#170]
Hi, 

I got this error during the building and generating the pdf : 

```
Successful building of ../output/ofBook.tex
! Undefined control sequence.
l.216 \tightlist

pandoc: Error producing PDF from TeX source
Error in building of ../output/ofBook.pdf
```

**Comment by bilderbuchi** [comment]
this kind of error previously meant that someplace above that statement, there was some illegal latex code created, possibly by some bad markdown in one of the chapters. as a starting point, check ofBook.tex if something jumps at you.

**Comment by mikewesthad** [comment]
I believe newer versions of latex have added a [definition](https://github.com/jgm/pandoc/issues/2485) for the tightlist command.  I updated the latex template we are using to include that definition (181228a3665b2829519f1982bd89fa5015cb3147).  Removing that build error reveals another problem, but I didn't have a chance to debug:

```
Successful building of ../output/ofBook.tex
! File ended while scanning use of \lst@temp.
<inserted text>
                \par
<*> ./tex2pdf.9516/input.tex

Output written on C:/Users/micha/Documents/GitHub/ofBook/scripts/tex2pdf.9516/i
nput.pdf (169 pages).
Transcript written on C:/Users/micha/Documents/GitHub/ofBook/scripts/tex2pdf.95
16/input.log.
Unable to update the static FcBlanks: 0x0600
Unable to update the static FcBlanks: 0x0601
Unable to update the static FcBlanks: 0x0602
Unable to update the static FcBlanks: 0x0603
Unable to update the static FcBlanks: 0x06dd
Unable to update the static FcBlanks: 0x070f
Unable to update the static FcBlanks: 0x2028
Unable to update the static FcBlanks: 0x2029
Unable to update the static FcBlanks: 0xfff9
Unable to update the static FcBlanks: 0xfffa
Unable to update the static FcBlanks: 0xfffb

pandoc.exe: Error producing PDF from TeX source
Error in building of ../output/ofBook.pdf
```

**Comment by mikewesthad** [comment]
Okay, it should be fixed now.

Looks like this merge (c71a9190564da77341251df1d25c25058de37210) re-added curly quotes into the game design chapter.  Re issue #69, I removed them again (7e917c29e6669ae63ead69b8e7eef61cde498f3c).

I also ran into issue #58 where `perspective_transform.jpg` from the project eva chapter is buggy.  I re-saved it with the same quality settings and everything works fine (6008d8a6e3239e8bf29c5cabbabff98f808a4a2e).
